myuw/static/css/mobile.less

```diff
@@ -645,14 +645,19 @@ label { font-weight: normal; margin-bottom: auto; }
                     display: block;
                     font-weight: bold;
                 /*line-height: 35px;*/ }
-                .card-badge-action {
+                .card-badge-action-btn { //change to button
```

Comment:
if you make this a peer to card-badge-container you can use this class in all cards and won't need to duplicate the class down below on line 996.

Reply:
Move the button out of the container under the card.